composer.json

```diff
@@ -16,7 +16,7 @@
         "ext-mongodb": "^1.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.36"
+        "phpunit/phpunit": "^6.4"
```

Comment:
This causes the 5.x jobs on Travis to fail, since there is no longer a compatible version of PHPUnit:

```
  Problem 1
    - phpunit/phpunit 6.5.6 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.5 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.4 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.3 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.2 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.1 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.5.0 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.4 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.3 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.2 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.1 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - phpunit/phpunit 6.4.0 requires php ^7.0 -> your PHP version (5.6.32) does not satisfy that requirement.
    - Installation request for phpunit/phpunit ^6.4 -> satisfiable by phpunit/phpunit[6.4.0, 6.4.1, 6.4.2, 6.4.3, 6.4.4, 6.5.0, 6.5.1, 6.5.2, 6.5.3, 6.5.4, 6.5.5, 6.5.6].
```

I'd suggest using `"^4.8.36 || ^6.4"`, which should support PHP 5.x while still allowing PHP 7 to use PHPUnit 6. That syntax is discussed in [Version Range in the Composer docs](https://getcomposer.org/doc/articles/versions.md#version-range).